File: Mobile_Development/Android/Udemy/CatSwitcher/app/src/main/java/com/example/catswitcher/MainActivity.java
package com.example.catswitcher;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    int flag=0;

    public void onClickChange(View view){
        ImageView image = (ImageView) findViewById(R.id.catImageView);
        if(flag==0){
            image.setImageResource(R.drawable.cat2);
            flag = 1;
        }
        else{
            image.setImageResource(R.drawable.cat1);
            flag=0;
        }
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Mobile_Development/Android/Udemy/KotlinCounter/app/src/main/java/com/example/kotlincounter/MainActivity.kt
package com.example.kotlincounter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var num=0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var countTextView = findViewById<TextView>(R.id.countTextView)

    }

    fun increase(view: View){
        num++
        val newCounter: String = "Count: $num"
        countTextView.setText(newCounter)
    }

    fun reset(view: View){
        num=0
        val newCounter: String = "Count: $num"
//        countTextView.setText(newCounter)
        countTextView.text = newCounter        // NEW WAY OF KOTLIN
    }
}
